components/src/dynamo/planner/tests/integration/test_actuation_knobs_live.py

```
@@ -0,0 +1,567 @@
+    ):
+        pods = connector.list_frontend_pods()
+        running = [p for p in pods if p.status.phase == "Running" and p.status.pod_ip]
+        if not running:
+            pytest.skip("No Running frontend pod with pod IP")
+        pod = running[0]
+
+        try:
+            model = connector.get_model_name(
+                require_prefill=False, require_decode=False
+            )
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)